units/Legion/Economy/legmohoconct.lua

```diff
@@ -0,0 +1,89 @@
+return {						--costs should be same as legmohocon and legmohoconin
+	legmohoconct = {
+		maxacc = 0,
+		maxdec = 4.5,
+		energycost = 8100,
+		metalcost = 640,
+		builddistance = 400,
+		builder = true,
+		buildpic = "LEGMOHOCON.DDS",
+		buildtime = 14100,
+		canassist = true,
+		canfight = true,
+		canguard = true,
+		canpatrol = true,
+		canreclaim = true,
+		canrepeat = false,
+		canstop = true,
+		cantbetransported = false,
+		collisionvolumeoffsets = "0 0 0",
+		collisionvolumescales = "31 32 31",
+		collisionvolumetype = "CylY",
+		usePieceCollisionVolumes = true,
+		energyupkeep = 30,
+		explodeas = "largeBuildingexplosiongeneric",
+		footprintx = 4,
+		footprintz = 4,
+		idleautoheal = 5,
+		idletime = 1800,
+		mass = 700,
+		health = 3900,
+		maxslope = 10,
+		maxwaterdepth = 0,
+		stealth = true,
```

Comment:
I believe you might be able to not have stealth, and then use `Spring.SetUnitLosMask` instead in the gadget to not have the ui show this as a stealthy unit to the player

Reply:
Couldn't get it to work, but wouldn't that still leave 2 dots for enemies to shoot at, even if the player sees only one?